src/classic/element/ReactElementValidator.js

```diff
@@ -298,9 +298,12 @@ function warnForPropsMutation(propName, element) {
 
   warning(
     false,
-    'Don\'t set .props.' + propName + ' of the React component' +
-    elementInfo + '. Instead, specify the correct value when ' +
-    'initially creating the element.' + ownerInfo
+    'Don\'t set .props.%s of the React component %s. ' +
+    'Instead, specify the correct value when ' +
+    'initially creating the element. %s',
+    propName,
+    elementinfo,
```

Comment:
Right will fix later on. 



On January 26, 2015 at 5:31:24 PM CST, Paul O’Shannessy <notifications@github.com> wrote:In src/classic/element/ReactElementValidator.js:  > @@ -298,9 +298,12 @@ function warnForPropsMutation(propName, element) { > > warning( > false, > - 'Don\'t set .props.' + propName + ' of the React component' + > - elementInfo + '. Instead, specify the correct value when ' + > - 'initially creating the element.' + ownerInfo > + 'Don\'t set .props.%s of the React component %s. ' + > + 'Instead, specify the correct value when ' + > + 'initially creating the element. %s', > + propName, > + elementinfo,   elementInfo  (is failing tests because of this)  —Reply to this email directly or view it on GitHub.